Scenes/HUD/UnitStatusCard.gd

```diff
@@ -0,0 +1,46 @@
+extends ButtonStatusCard
+
+
+@export var _level_panel: ShortResourceStatusPanel
+
+
+func _ready():
+	SelectUnit.selected_unit_changed.connect(_on_selected_unit_changed)
+
+
+func _on_selected_unit_changed(prev_unit):
+	var selected_unit = SelectUnit.get_selected_unit()
+	
+	match [selected_unit != null, prev_unit != null]:
```

Comment:
I don't think this is working as intended. The third branch will never be run because the first two match cases capture all possible cases.

Replace with something simpler that will work for sure.

```
if prev is not null and is connected => disconnect level_changed

if current is not null
   set visible, set pressed, connect level_changed, etc
else
   disable visiblem set pressed false
```

Reply:
Done